BE/src/main/java/com/todolist/controller/WorkController.java

```diff
@@ -1,19 +1,26 @@
 package com.todolist.controller;
 
 import com.todolist.dto.ColumnListDto;
+import com.todolist.dto.ModifiedWorkDto;
+import com.todolist.dto.WorkDeletionDto;
 import com.todolist.dto.WorkDto;
 import com.todolist.dto.WorkCreationDto;
+import com.todolist.dto.WorkModificationDto;
 import com.todolist.dto.WorkMovementDto;
 import com.todolist.service.WorkService;
 
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@RequestMapping("works")
```

Comment:
저는 구분자까지 명시적으로 아래와 같이 작성하는 편을 선호합니다.

```suggestion
@RequestMapping("/works")
```